### PR TITLE
add file README.pt-br.md and translating

### PR DESCRIPTION
**Pull request by Simplesmente**
Hi, my name is André Teles, I am tried translate it documentation to my language. I want colaborate with your project the way that i can.

### CODE REVIEW

File: README.pt-br.md [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+phpbrew contrói e instala multiplas versões do php no seu diretório $HOME.
+
+Oque phpbrew pode fazer para você:
+
+- As configurações são feitas dentro de variaveis, não se preocupe mais com caminhos.
+- Construa php com diferentes variaveis como PDO, mysql, sqlite, debug etc...  
+- Compile modulos php do apache e separe eles por diferentes versões.
+- Construa e instale php(s) no seu diretório home, assim não irá precisar de permissão de root.
+- Troque de versões muito facilmente pois é integrado com bash/zsh shell.
+- Detecção automática de recurso.
```
Comment (marcioAlmada) [inline README.pt-br.md]: Instead of "recurso", how about "código fonte"?

File: README.pt-br.md [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+- Instale & abilite extenssões php no ambiente atual com facilidade.
```
Comment (marcioAlmada) [inline README.pt-br.md]: typo "extensões"

File: README.pt-br.md [hunk 3]
```diff
@@ -0,0 +1,566 @@
...
+=======
+
+*Leia este documento em outras linguaguens: [English](README.md), [Português - BR](README.pt-br.md), [日本語](README.ja.md).*
+
+[![Build Status](https://travis-ci.org/phpbrew/phpbrew.svg?branch=master)](https://travis-ci.org/phpbrew/phpbrew)
+[![Coverage Status](https://img.shields.io/coveralls/phpbrew/phpbrew.svg)](https://coveralls.io/r/phpbrew/phpbrew)
+
+phpbrew contrói e instala multiplas versões do php no seu diretório $HOME.
+
+Oque phpbrew pode fazer para você:
```
Comment (marcioAlmada) [inline README.pt-br.md]: missing a space here "O que" :)

**Comment by marcioAlmada** [comment]
Thanks for starting this!

**Comment by Simplesmente** [comment]
So, I am starting now the contribute with projects opensource, for learn plus about how contribute. But if you want I will go send pull request, If you think is useful i go be happy.

**Comment by c9s** [comment]
@Simplesmente welcome!

**Pull request merged**